Lib/test/test_typing.py

```diff
@@ -4952,6 +4952,7 @@ def barfoo(x: AT): ...
         def barfoo2(x: CT): ...
         self.assertIs(get_type_hints(barfoo2, globals(), locals())['x'], CT)
 
+    @unittest.expectedFailure # TODO: RUSTPYTHON; 'List[list["C2"]]' != "List[list['C2']]"
```

Comment:
is there any option to change the quote style?

Reply:
No:/ not from the `Generator` directly, I think that this needs to be fixed upstream (over Ruff's side)

Reply:
Great, thank you so much!